### PR TITLE
Add SOAP vs REST text

### PR DESCRIPTION
**Pull request by Sander3003**
This should address user/developer questions around this topic.

### CODE REVIEW

**Comment by Sander3003** [comment]
@pczeeuw can you do a review?

File: Architecture/Platform-components-description.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ For asynchronous web services the response contains a correlation ID. This Corre
 
 Furthermore each SOAP message has a header which contains the user's organisation ID. This table displays an overview of the WSDL's including operations and fields in the request and response objects.
 
+SOAP vs REST
+SOAP is chosen in the open smart grid platform webservices over REST for the following reasons:
+- REST is resources/data oriented (put, get, delete) while the open smart grid platform is function/method oriented
+- SOAP has the advantage of having a contract (WSDL)
+- SOAP has extensive security features that are being used in the open smart grid platform to meet the high security demands/requirements requested by e.g. the energy utilities
+- Energy companies are generally not progressive in terms of technology. SOAP is acceptable for energy companies and REST is sometimes seen as new and insecure.
+The benefits of REST (e.g. speed / less overhead) does not outweigh the benefits of SOAP. More general information on this topic can be found [online](http://spf13.com/post/soap-vs-rest). Contributions to make REST available on the open smart grid platform are welcome.
```
Comment (pczeeuw) [inline Architecture/Platform-components-description.md]: Contributions to make REST available on the open smart grid platform are welcome.

Maybe drop this line? Or place it in chapter 2.5, and make a reference to it.

Comment (Sander3003) [inline Architecture/Platform-components-description.md]: I was in doubt about this. It should be pretty clear that we want to make this an community open source project. I will remove this line.

Comment (pczeeuw) [inline Architecture/Platform-components-description.md]: I'll add in to chapter 2.5

**Pull request merged**